kde-plasma/kinfocenter/kinfocenter-9999.ebuild

```diff
@@ -34,7 +34,6 @@ DEPEND="
 	usb? ( virtual/libusb:1 )
 "
 RDEPEND="${DEPEND}
-	dev-qt/qdbus:*
```

Comment:
`>=dev-qt/qttools-${QTMIN}:6[qdbus]` is not needed? Did you check that?

Reply:
I verified that qdbus binary is necessary and added the missing qttools:6[qdbus] as preferred option to satisfy it.